### PR TITLE
Added avatars to chat

### PR DESCRIPTION
**Pull request by TheHollidayInn**

### CODE REVIEW

File: website/client/components/chat/chatMessages.vue [hunk 1]
```diff
@@ -145,7 +154,29 @@ export default {
...
+      let promises = [];
+
+      this.messages.forEach(message => {
+        let uuid = message.uuid;
+        if (!this.cachedProfileData[uuid]) {
+          promises.push(axios.get(`/api/v3/members/${uuid}`));
+        }
+      });
+
+      let results = await Bluebird.all(promises);
```
Comment (paglias) [inline website/client/components/chat/chatMessages.vue:174]: You should be able to use Promise.all, without loading Bluebird

Comment (TheHollidayInn) [reply website/client/components/chat/chatMessages.vue:174]: Is there a difference? I'll switch to Promise.all, but I am just curious.

Comment (paglias) [reply website/client/components/chat/chatMessages.vue:174]: No difference but I think it's the first time I've seen bluebird used in the client so if we avoid loading it is one less dependency

Comment (TheHollidayInn) [reply website/client/components/chat/chatMessages.vue:174]: Sounds good to me!

**Pull request merged**